Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+Artificial Intelligence is being used to drastically **[reduce toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/)**, greatly improving ESDC's commitment to citizens through its continuously improving service standards, enabling auto-enrolment by design, providing insights for service innovation, while ensuring ethical use of [automated decision-making](https://tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592). Artificial Intelligence is a shared expertise between the Chief Information Officer (CIO), responsible for the technology infrastructure hosting AI software, and the Chief Data Officer (CDO), responsible for data sciences.
+
+All of ESDC's services are connected and interoperable to enable the **"tell us once"** goal.
+All of ESDC's services use a single account, tied to a single identity, in order to avoid multiple separate logins and use **[modern identity verification](https://gcn.com/articles/2019/03/25/mobile-verification.aspx)** to drastically improve identity theft protection for citizens.
+
+## 2. An open, collaborative, and accessible government
+
+The principles of open by default and inclusive by design are integrated into all new processes, systems, solutions, and services.
+
+ESDC **publishes a large majority of its source code under an open source licence**, on the Internet, and seeks contribution from other departments, other jurisdictions, and the public.
+Only on exceptional, well documented and justified cases, is source code not published.
+By releasing its source code under an open source licence as well as its data under the open government licence, **ESDC's IT community is not restrained to ESDC staff only**.
+ESDC leverages crowd sourcing by including other departments, other jurisdictions, and the public to contribute back to its IT solutions.
+This [promotes canadian economic growth, skills development in Canada](https://www.theglobeandmail.com/business/commentary/article-the-essentials-for-canadas-digital-economy/), and improves trusts with canadians as well as greatly increasing ESDC's IT expertise from such broad community relationships.
+
+ESDC **publishes a large majority of the data it uses to make its decisions as open data sets under the Open Government Licence**, promoting transparency and trusts with citizens, and enabling the broader public to make use of government data for developing tools and products that contribute to their well-being and economic growth. This does not mean publishing sensitive data. Only on exceptional, well documented and justified cases, is data not published as open data.
+
+ESDC's policy and program development are done "in the open", over the Internet, enabling active participation with stakeholders whether they be citizens, 3rd party partners like banks, employers, or other jurisdictions.
+
+Technology choices **adopt open standards with preference for Open Source solutions** in order to avoid vendor lock-in, enable interoperability, and collaboration.
```

Comment:
Prepend with "Following the [Directive on Management of IT](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=15249)," ?